extensions/amp-serviceworker-installation/amp-serviceworker-installation.md

```diff
@@ -0,0 +1,48 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### <a name="amp-serviceworker-installation"></a> `amp-serviceworker-installation`
+
+The `amp-serviceworker-installation` component allows installing a [ServiceWorker](http://www.html5rocks.com/en/tutorials/service-worker/introduction/) for the current page.
```

Comment:
suggest changing to https://github.com/slightlyoff/ServiceWorker/blob/master/explainer.md